python/tvm/relay/op/strategy/x86.py

```diff
@@ -175,21 +175,10 @@ def conv2d_strategy_cpu(attrs, inputs, out_type, target):
     else:  # group_conv2d
         if layout == "NCHW":
             assert kernel_layout == "OIHW"
-            if not is_auto_scheduler_enabled():
-                logger.warning("group_conv2d is not optimized for x86 with autotvm.")
-            strategy.add_implementation(
-                wrap_compute_conv2d(topi.nn.group_conv2d_nchw, has_groups=True),
-                wrap_topi_schedule(topi.generic.schedule_group_conv2d_nchw),
-                name="group_conv2d_nchw.generic",
-            )
```

Comment:
What kind of strategy logic would include both?  Having this code on an `else` branch of `if not is_auto_scheduler_enabled():`?  Or something else?

Reply:
`if not is_auto_scheduler_enabled()`: ... We prefer auto scheduler be the default stragegy.

Reply:
Current code should now be able to run with auto scheduler by running with `"relay.backend.use_auto_scheduler": True`, and otherwise using my schedule.